ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientBase.java

```diff
@@ -2,6 +2,7 @@
 
 import com.microsoft.azure.kusto.data.exceptions.ExceptionUtils;
 import com.microsoft.azure.kusto.ingest.source.CompressionType;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
are you using this import?

Reply:
Via `StringUtils.isEmpty()` on line 58

Reply:
Right now there is a PR #415 that removes this dep entirely.

Can you use `CoreUtils.isNullOrEmpty`?